src/current.cpp

```diff
@@ -244,7 +248,7 @@ void current_check_stale()
 {
     try
     {
-        if (millis() - current_last_hash > 200000)
+        if (millis() - current_last_hash > 600000)
         {
             l_error(TAG_CURRENT, "No hash received in the last 3 minutes. Restarting...");
```

Comment:
@KillerInk 10min waiting is not too much?
If fine, worth update the error with the proper amount of time

Reply:
no its fine. pool.io increase diff to 0.001 and there can be hashtimes longer then 5min where also no new job trigger.
and after 10min then min there should be latest a new job request(new block). if not ,connection realy died.